.github/workflows/gradle.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
       - name: Run Standard Source Tests (Master)
         if: success() && github.ref == 'refs/heads/master'
-        run: ./gradlew --no-daemon standardSourceTest --scan
+        run: ./gradlew --no-daemon standardSourceTestPython --scan
```

Comment:
is it intentional to remove `standardSourceTest`? I guess since everything is python at the moment.

Reply:
yeah. there's more crunching of all the gradle code that needs to happen. for now all the standard source tests are run as python. all this can be cleaned up later.